common/djangoapps/coursetalk/models.py

```diff
@@ -0,0 +1,29 @@
+""" Models for CourseTalk configurations """
+from __future__ import unicode_literals
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from config_models.models import ConfigurationModel
+
+
+class CourseTalkWidgetConfiguration(ConfigurationModel):
+    """
+    Enable CourseTalk widget on course page
+    """
+    platform_key = models.fields.CharField(
+        max_length=50,
+        help_text=_(
+            "This key needs to associate CourseTalk reviews with your platform. "
+            "Better to use domain name "
+            "Ex: for 'http://edx.org' platform_key will be 'edx'"
```

Comment:
I find this text a little confusing. Maybe something like, "This is is needed to associate CourseTalk widgets with your platform. It is best to use a domain name. For example, if your platform is `http://edx.org`, the `platform_key` would be `'edx'`."